simplified-app-vanilla/src/main/java/org/nypl/simplified/vanilla/VanillaDocumentStoreConfiguration.kt

```diff
@@ -0,0 +1,26 @@
+package org.nypl.simplified.vanilla
+
+import org.librarysimplified.documents.DocumentConfiguration
+import org.librarysimplified.documents.DocumentConfigurationServiceType
+import java.net.URI
+
+class VanillaDocumentStoreConfiguration : DocumentConfigurationServiceType {
+
+  override val privacyPolicy: DocumentConfiguration? =
+    null
+
+  override val about: DocumentConfiguration? =
+    DocumentConfiguration(
+      name = "about.html",
+      remoteURI = URI.create("http://localhost/about.html")
```

Comment:
This is just until we have this document actually hosted somewhere. Could probably host it in the GitHub repository.

Reply:
I feel like SimplyE had one and now doesn't show it... We really need to redo the whole set of documents.